controllers/akodeploymentconfig/cluster/cluster_controller_unit_test.go

```diff
@@ -39,21 +40,32 @@ loadBalancerAndIngressService:
             cluster_name: -test-cluster
             cni_plugin: ""
             sync_namespace: ""
+            enable_EVH: ""
+            layer_7_only: ""
+            services_api: ""
+            namespace_selector:
+                label_key: ""
+                label_value: ""
         network_settings:
             subnet_ip: 10.0.0.0
             subnet_prefix: "24"
             network_name: test-akdc
             node_network_list: '[{"networkName":"test-node-network-1","cidrs":["10.0.0.0/24","192.168.0.0/24"]}]'
             vip_network_list: '[{"networkName":"test-akdc"}]'
+            enable_rhi: ""
+            bgp_peer_labels: ""
         l7_settings:
             disable_ingress_class: true
-            default_ing_controller: true
+            default_ing_controller: false
```

Comment:
should this be interpreted as `should we use the default ingress controller`? maybe change to `use_default_ing_controller`, otherwise a bool here is not expected

Reply:
This should be interpreted as `should AKO be the default ingress controller`, so normally we set this to be false.

Reply:
that makes sense to me now!